WordPress/WordPressTest/NotificationUtility.swift

```diff
@@ -1,51 +1,50 @@
-
+import CoreData
 import Foundation
 import XCTest
 @testable import WordPress
 
 class NotificationUtility {
```

Comment:
_Out of scope_: When first seeing this used in the tests, I thought `NotificationUtility` was a production code type, whereas it is a "utility" to setup state and get data in the test code. 

Have you got any idea on a name that would make its role clearer?

`NotificationTestingUtility` is an easy choice, but it seems too vague. All methods load data from JSON fixtures. Maybe `NotificationFixturesLoader` would be a good name?

Reply:
Having "Test" in the name definitely helps clarifying its purpose. It's probably also worth exploring a general approach to load data from JSON fixture.